quiz q3 solution.txt

```diff
@@ -0,0 +1,39 @@
+bca_info = """BCA is Three Year Course Regarding Computer Science"""
+
+with open('bca.txt','w') as bca:
+    bca.write(bca_info)
+
+A = 0
+B = 0
+C = 0
+
+from csv import reader
+
+with open('intro.txt','r' ) as intro:
+    bca = intro.readlines()
+
+    for i in bca:
+        stripped_i = i.strip()

```

Comment:
can you please explain the meaning of stripped and why we used it??

Reply:
Thank you for Commenting ! So, stripped_i is just an object The "strip()" is used for Remove any Whitespace if there is any Because If there's any whitespace then It Won't Count A,B,C As First Character. I Hope you get it.

Reply:
ok thank you now i understand it but i have one confusion what is the whitespace can u please explain??

Reply:
The Simple Meaning of Whitespace is It is Space Ahead of The String.

Reply:
ok now i understand thank you for explaination.